dev-requirements.txt

```diff
@@ -1,7 +1,7 @@
-git+https://github.com/dbt-labs/dbt-adapters.git@main
-git+https://github.com/dbt-labs/dbt-adapters.git@main#subdirectory=dbt-tests-adapter
+git+https://github.com/dbt-labs/dbt-adapters.git@snapshot_column_names
```

Comment:
I know it's obvious, but I'm just leaving a reminder that these changes will need to be left out of the merge.